kata.py

```diff
@@ -12,18 +19,34 @@ def Empty(string):
                 else:
                     answer = 0
                     for num in string[4:].split('\n'):
+                        Negative(num)
                         answer += int(num)
-                    return answer
+                    if len(negatives_list) == 0:
+                        return answer
+                    else:
+                        raise Exception(f'Negatives not allowed: {negatives_list}')
             elif Empty(string.split('\n')[1]):
                 return 0
             else:
                 answer = 0
                 delimiter = string.split('\n')[0].strip('//')
                 for num in string.split('\n')[1].split(delimiter):
+                    Negative(num)
```

Comment:
To follow on from a message on a previous commit, this is probably _another_ use case of this `Add` function now doing even more work and finding even more responsibilities. I think extracting into a `Negative` function is good, but I think there are probably some refactoring opportunities to take this a little further 😄